From 69e12ea12036545021e620f5bcc7034346fecaa3 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Sun, 9 Jun 2019 09:57:11 -0600
Subject: [PATCH 2/3] openssh: Update Android build for v8.0p1

* Drop removed files from Android.mk
* Add new functional files to libssh in Android.mk

Change-Id: I8d35d8e16e06996a9a5a8cdf6efcd350f5d8d7bc
---
 Android.mk | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/Android.mk b/Android.mk
index 98ac9b13..268930ec 100644
--- a/Android.mk
+++ b/Android.mk
@@ -37,13 +37,12 @@ LOCAL_SRC_FILES := \
     hostfile.c \
     kex.c \
     kexc25519.c \
-    kexc25519c.c \
     kexdh.c \
-    kexdhc.c \
     kexecdh.c \
-    kexecdhc.c \
+    kexgen.c \
     kexgex.c \
     kexgexc.c \
+    kexsntrup4591761x25519.c \
     krl.c \
     log.c \
     mac.c \
@@ -53,7 +52,6 @@ LOCAL_SRC_FILES := \
     monitor_fdpass.c \
     msg.c \
     nchan.c \
-    opacket.c \
     openbsd-compat/bcrypt_pbkdf.c \
     openbsd-compat/bindresvport.c \
     openbsd-compat/blowfish.c \
@@ -102,6 +100,7 @@ LOCAL_SRC_FILES := \
     sshkey-xmss.c \
     ssh-rsa.c \
     ssh-xmss.c \
+    sntrup4591761.c \
     ttymodes.c \
     uidswap.c \
     umac128.c \
@@ -249,9 +248,6 @@ LOCAL_SRC_FILES := \
     groupaccess.c \
     gss-serv-krb5.c \
     gss-serv.c \
-    kexc25519s.c \
-    kexdhs.c \
-    kexecdhs.c \
     kexgexs.c \
     loginrec.c \
     md5crypt.c \
-- 
2.17.1

